Bukkit/src/main/java/com/plotsquared/bukkit/listener/BlockEventListener.java

```diff
@@ -1224,18 +1210,9 @@ public void onBlockMultiPlace(BlockMultiPlaceEvent event) {
             if (pp.hasPermission(Permission.PERMISSION_ADMIN_BUILD_HEIGHT_LIMIT)) {
                 continue;
             }
-            if (currentLocation.getY() >= area.getMaxBuildHeight() || currentLocation.getY() < area.getMinBuildHeight()) {
-                pp.sendMessage(
-                        TranslatableCaption.of("height.height_limit"),
-                        TagResolver.builder()
-                                .tag("minheight", Tag.inserting(Component.text(area.getMinBuildHeight())))
-                                .tag("maxheight", Tag.inserting(Component.text(area.getMaxBuildHeight())))
-                                .build()
-                );
-                if (area.notifyIfOutsideBuildArea(pp, currentLocation.getY())) {
-                    event.setCancelled(true);
-                    break;
-                }
+            if (area.notifyIfOutsideBuildArea(pp, currentLocation.getY())) {
+                event.setCancelled(true);
+                break;
             }
```

Comment:
The check in line `1227` looks exactly like the check in `notifyIfOutsideBuildArea()`.

```
# line `1227`
currentLocation.getY() >= area.getMaxBuildHeight() || currentLocation.getY() < area.getMinBuildHeight()
```

[/Core/src/main/java/com/plotsquared/core/plot/PlotArea.java#L676-L677](https://github.com/RedstoneFuture/PlotSquared/blob/79cc9626750686f03fb6b5d5690bfc6ced9f9569/Core/src/main/java/com/plotsquared/core/plot/PlotArea.java#L676-L677)
```java
    public boolean notifyIfOutsideBuildArea(PlotPlayer<?> player, int y) {
        if (!buildRangeContainsY(y) && !player.hasPermission(Permission.PERMISSION_ADMIN_BUILD_HEIGHT_LIMIT)) {
            ...
            // Return true if "failed" as the method will always be inverted otherwise
            return true;
        }
        return false;
    }
```